src/hotspot/share/cds/dynamicArchive.cpp

```diff
@@ -158,6 +162,11 @@ class DynamicArchiveBuilder : public ArchiveBuilder {
     write_archive(serialized_data);
     release_header();
 
+    post_dump();
+
+    // Restore dumptime tables
+    SystemDictionaryShared::restore_dumptime_tables();
```

Comment:
Maybe this should be put inside post_dump as well?

Reply:
It would be better to keep it here with SystemDictionaryShared::clone_dumptime_table().